game/src/contrib/entities/EntityFactory.java

```diff
@@ -40,8 +41,8 @@ public class EntityFactory {
      */
     public static Entity newHero() throws IOException {
         final int fireballCoolDown = 2;
-        final float xSpeed = 0.3f;
-        final float ySpeed = 0.3f;
+        final float xSpeed = Constants.X_AXIS_SPEED_HERO;
```

Comment:
prinzipiell find ich das gu => Magic Numbers sind doof.
Aber das muss keine Konstante in `Constants` sein, die kann in der `EntityFactory` als. `private static final` angelegt werden.

Reply:
Ok wird geändert.